pipestat/html_reports_pipestat.py

```diff
@@ -937,12 +945,20 @@ def get_file_for_project(prj, pipeline_name, appendix=None, directory=None, repo
     Format of the output path:
     {output_dir}/{directory}/{p.name}_{pipeline_name}_{active_amendments}_{appendix}
 
-    :param looper.Project prj: project object
+    :param pipestat manager object prj: project object
```

Comment:
Use the correct type here, not a text description of it.

Reply:
also this is confusing, is it a `pipestat.PipestatManager` ? or a Project object? it says both.